xml/libvirt_migrating_vms.xml

```diff
@@ -0,0 +1,623 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+xmlns:xi="http://www.w3.org/2001/XInclude"
+xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+xml:id="sec-libvirt-admin-migrate">
+  <title>Migrating &vmguest;s</title>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker></dm:bugtracker>
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+  </info>
+  <para>
+    One of the major advantages of virtualization is that &vmguest;s are
+    portable. When a &vmhost; needs maintenance, or when the host becomes
+    overloaded, the guests can be moved to another &vmhost;. &kvm; and &xen;
+    even support <quote>live</quote> migrations during which the &vmguest; is
+    constantly available.
+  </para>
+  <sect1 xml:id="libvirt-migration-types">
+    <title>Types of migration</title>
+
+    <para>
+      Depending on the required scenario, there are three ways you can migrate
+      virtual machines (VM).
+    </para>
+
+    <variablelist>
+      <varlistentry>
+        <term>Live migration</term>
+        <listitem>
+          <para>
+            The source VM continues to run while its memory is transferred to
```

Comment:
Here we say the VM memory is transferred. Under the "non-live" section, we say VM configuration and state are transferred. We should be consistent in these sections. How about "VM configuration and memory"?

Related, we use "source", "destination", and "target" consistently. E.g. in the Live section we talk about "source VM", "target VM", and "target host". In Non-live, we say "destination VM".  We should ensure consistent migration terms throughout the docs.